### PR TITLE
Public Cloud: Fix the 'ip' command

### PR DESCRIPTION
**Pull request by pdostal**
* Hotfix to #18123 
 * Verification runs: [pdostal-server.suse.cz](https://pdostal-server.suse.cz)

### CODE REVIEW

File: tests/publiccloud/instance_overview.pm [hunk 1]
```diff
@@ -33,6 +33,7 @@ sub run {
 
     assert_script_run("ps aux | nl");
 
+    my $ip_color = (is_sle('>=15-SP3')) ? '-c=never' : '';
     assert_script_run("ip -c=never a s");
```
Comment (m-dati) [inline tests/publiccloud/instance_overview.pm:37]: Is it probably missing use of that $`ip_color` parameter?

**Review by alvarocarvajald — APPROVED** [review]
LGTM

**Review by m-dati — APPROVED** [review]
Thanks. LGTM

**Review by ricardobranco777 — APPROVED** [review]

**Review by mpagot — COMMENTED** [review]
LGTM

**Pull request merged**